### PR TITLE
Nass errors on nass page 186329

### PR DESCRIPTION
**Pull request by HarrisonShawAdvania**
Errors that should be on the Nass Page are now shown on the Nass page instead of sending the user back to Enter_Details page

### CODE REVIEW

**Review by marten-wetterberg — CHANGES_REQUESTED** [review]
Is there any chance we could get a unit/cypress test to cover this as a regression test?

**Comment by HarrisonShawAdvania** [comment]
@marten-wetterberg I can work on some unit tests and work with Sam on cypress tests.

**Comment by HarrisonShawAdvania** [comment]
@marten-wetterberg cypress tests added to cover the new error messages

File: tests/cypress/support/commands.ts
Comment (samfallowfield) [inline tests/cypress/support/commands.ts:1]: This doesn't need to be broken out into a separate function, only has one current use and will need to be changed after upcoming refactor

**Review by marten-wetterberg — APPROVED** [review]

**Pull request merged**